lang/python/django2-cors-headers/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=django2-cors-headers
```

Comment:
if you rename `django2` to `django` this looks pretty ready and can be split into another PR

Reply:
Thank you, done [that](https://github.com/openwrt/packages/pull/10374).